plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/Library.java

```diff
@@ -26,4 +38,38 @@ public String getId() {
   public IPath getContainerPath() {
     return new Path(CONTAINER_PATH_PREFIX + "/" + id);
   }
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public URI getSiteUri() {
+    return siteUri;
+  }
+
+  public void setSiteUri(URI siteUri) {
+    this.siteUri = siteUri;
+  }
+
+  public List<LibraryFile> getLibraryFiles() {
+    return libraryFiles;
+  }
+
+  public void setLibraryFiles(List<LibraryFile> libraryFiles) {
+    if (libraryFiles != null) {
+      this.libraryFiles = new LinkedList<>(libraryFiles);
+    }
+  }
+
+  public String getDescription() {
```

Comment:
I'm personally not a big fan of these kind of methods, except when encoded in a `toString()` as it's not clear from the signatures alone whether this class has 2 or 3  attributes: `id`, `name`, and `description`.  This kind of logic is better in the presentation layer.